epicweb-dev/epic-stack
# Andrecasal/issue94

Fixes #94

Installs Playwright browsers on setup.

## Test Plan

1. Run with local folder's epic-stack: npx create-remix@latest --typescript --install --template ./epic-stack
2. Check the Playwright browsers where installed ✅

## Checklist

N/A

## Screenshots

N/A
